sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -583,6 +611,24 @@ case class MyParser(spark: SparkSession, delegate: ParserInterface) extends Pars
     delegate.parseQuery(sqlText)
 }
 
+case class AddTagRule() extends Rule[SparkPlan] {
```

Comment:
`AddTagRule` looks too general in a long term perspective. Shall we use more specific one like `ExplainPlanInfoTagRule`?